### PR TITLE
feat: react trial logs [DET-3128]

### PR DESCRIPTION
**Pull request by hkang1**
## Description

Screen Shot 2020-07-06 at 1 52 45 PM

## Test Plan

This page will be accessible from the WebUI once this [ticket](https://determinedai.atlassian.net/browse/DET-3530) is done. For now it is accessible directly `/det/trials/<trial_id>/logs`.

## Commentary (optional)

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

**Comment by hamidzr** [comment]
is this using the new streaming endpoint?

**Comment by hkang1** [comment]
> is this using the new streaming endpoint?

This is using the older existing endpoint. Was thinking I should wait for [this](https://determinedai.atlassian.net/browse/DET-3451) to land.

File: webui/react/src/pages/TrialLogs.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+import { TrialLogsParams } from 'services/types';
+import { Log } from 'types';
+
+interface Params {
+  trialId: string;
+}
+
+const TAIL_SIZE = 10000;
+
+const TrialLogs: React.FC = () => {
```
Comment (hamidzr) [inline webui/react/src/pages/TrialLogs.tsx:19]: question: the logic and this component looks a lot like the master logs code. did you want to explicitly generalize it in another ticket/PR?

**Comment by hamidzr** [comment]
question: the idea is that this would theoretically support command logs as well?

**Comment by hamidzr** [comment]
approach # 1 : refactor master logs => add the new trial/command logs component
approach # 2 : add the new trial logs component => refactor master logs and trial/command logs (edited) 

# 2 which this seems to be doing can result in a better refactor so that can be the better option if we quickly refactor before they diverge

**Comment by hkang1** [comment]
> is this using the new streaming endpoint?

That will be a separate ticket that will follow suit.

**Comment by hkang1** [comment]
> Works great 🎉 ! I think there is a good opportunity to reuse the master logs code here without duplicating it otherwise these could diverge quickly.
> 
> style nit: in general I think the log viewers could use a bit higher contrast
> Screen Capture_select-area_20200707162929

Good eye, I looked at the source and the log viewer was missing a default text color!

**Pull request merged**